src/edu/ucsb/nceas/metacat/properties/BackupPropertiesDelegate.java

```diff
@@ -0,0 +1,284 @@
+package edu.ucsb.nceas.metacat.properties;
+
+import edu.ucsb.nceas.metacat.service.ServiceService;
+import edu.ucsb.nceas.metacat.shared.MetacatUtilException;
+import edu.ucsb.nceas.metacat.shared.ServiceException;
+import edu.ucsb.nceas.metacat.util.SystemUtil;
+import edu.ucsb.nceas.utilities.GeneralPropertyException;
+import edu.ucsb.nceas.utilities.MetaDataProperty;
+import edu.ucsb.nceas.utilities.PropertiesMetaData;
+import edu.ucsb.nceas.utilities.PropertyNotFoundException;
+import edu.ucsb.nceas.utilities.SortedProperties;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import javax.xml.transform.TransformerException;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.InvalidPathException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+import java.util.Set;
+import java.util.Vector;
+
+public class BackupPropertiesDelegate {
+    static boolean bypassAlreadyChecked = false;
+    // Full Path to the DIRECTORY containing the configuration BACKUP files (such as
+    // metacat.properties.backup etc.). Example: /var/metacat/.metacat
+    static Path backupDirPath;
+    private static SortedProperties mainBackupProperties = null;
+
+    private static BackupPropertiesDelegate instance = null;
+    String mainBackupFilePath = null;
+    private static final Log logMetacat = LogFactory.getLog(PropertiesWrapper.class);
+
+
+    private BackupPropertiesDelegate()
+        throws GeneralPropertyException {
+
+        if (Files.exists(getBackupDirPath())) {
+            try {
+                // write the file to ~/.metacat that contains the real path to the backup directory
+                SystemUtil.writeStoredBackupFile(getBackupDirPath().toString());
```

Comment:
Is this truly still `~/.metacat`? That has always seemed like a fragile location. And what became of our discussion of this backup no longer being needed with the new site properties file which doesn't get overwritten>

Reply:
I agree that ultimately, it makes sense to eliminate the old "backups" scheme and replace it with the new site properties approach. For now, they exist in parallel and in harmony. 

Getting rid of "backups" isn't trivial, though, so I think it would be prudent to attack as a separate PR once this one has been merged and has had the chance to cook for a bit.

Have created an issue for this: #1628 -- Replace Old "Backup Properties" scheme with new "Site Properties" Approach

Reply:
to answer the first question - yes, the code still uses ~/.metacat, but not to stores the backups - this stores a text file that contains the path to the backups (see [`SystemUtil.getStoredBackupDir()`](https://github.com/NCEAS/metacat/blob/4975c3b6301127fc93c54b32a75bd847101a02ae/src/edu/ucsb/nceas/metacat/util/SystemUtil.java#LL434C23-L434C41)
```
// Check if there is a file at
// <user_home>/<application_context>/backup-location. If so, it
// should contain one line that is a file that points to a writable
// directory. If that is true, use that value as the backup dir.
```

This does sound fragile (particularly since the `tomcat` user doesn't always have a home directory), but it's not obvious to me what would be a better approach (for old-skool, non-containerized deployments). We can have some good conversations about this when the time comes!